### PR TITLE
Safari CSS Webkit Issue

### LINKED ISSUE #2634: Fix removal suggestion styles not appearing on safari
What?
On safari, our `text-decoration: line-through` styling for removal suggestions doesn't work. We should use the right property for safari and make sure the styles show up on all the browsers we target. 

Initial estimate
1

### PR DESCRIPTION
**Pull request by qweliant**
## Issue(s) Resolved
#2634 

## Test Plan
Visit Safari browser and another browser and view http://localhost:9876/pub/pegl1ofe/draft
be sure the text looks like below in both browsers

## Screenshots (if applicable)
image
image

### Notes/Context/Gotchas
the neater one line webkit solution did not work so i split them up

### Supporting Docs
https://stackoverflow.com/questions/51181010/text-decoration-line-through-css-does-not-work-on-safari-with-my-completed-cla

### CODE REVIEW

**Comment by qweliant** [comment]
Should i go through and test any other potential css bugs from the text-decorator?

File: client/components/Editor/styles/suggestedEdits.scss [hunk 1]
```diff
@@ -43,7 +43,12 @@ span[data-suggestion-mark='true'][data-suggestion-kind='addition'] {
 }
 
 span[data-suggestion-mark='true'][data-suggestion-kind='removal'] {
-	text-decoration: 2px $removal line-through;
+	text-decoration: 2px;
+	-webkit-text-decoration: 2px;
```
Comment (kalilsn) [inline client/components/Editor/styles/suggestedEdits.scss:47]: safari doesn't seem to recognize this property, so the line is 1px. however, it looks like there's wide support for `text-decoration-thickness`, including unprefixed on safari

**Review by kalilsn — CHANGES_REQUESTED** [review]
i wouldn't spend time looking for css bugs right now, but if you noticed anything else broken then let me know and open an issue!

File: client/components/Editor/styles/suggestedEdits.scss (hunk 1 above)
Comment (qweliant) [reply client/components/Editor/styles/suggestedEdits.scss:47]: oof, thought i copied from the stash on master

**Review by kalilsn — APPROVED** [review]
Looks great!

**Pull request merged**